api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -225,8 +226,10 @@ public DbUser updateProfile(
 
     if (!user.getGivenName().equalsIgnoreCase(updatedProfile.getGivenName())
         || !user.getFamilyName().equalsIgnoreCase(updatedProfile.getFamilyName())) {
-      userService.setDataUserCodeOfConductNameOutOfDate(
-          updatedProfile.getGivenName(), updatedProfile.getFamilyName());
+      DbUserCodeOfConductAgreement duccAgreement = user.getDuccAgreement();
+      if (duccAgreement != null) {
```

Comment:
unlikely that the user would update their profile before agreeing to the DUCC, but it may be possible